hapi-fees/pom.xml

```diff
@@ -10,7 +10,7 @@
   <parent>
     <groupId>com.hedera.hashgraph</groupId>
     <artifactId>hedera-services</artifactId>
-    <version>0.11.1-SNAPSHOT</version>
+    <version>0.13.0-SNAPSHOT</version>
```

Comment:
Will these changes be not committed automatically in the next stage of release process?